cnxepub/adapters.py

```diff
@@ -193,6 +194,15 @@ def _make_package(binder):
                 resource = resources.get(filename)
                 if resource:
                     reference.bind(resource, '../resources/{}')
+                else:
+                    if reference.uri.startswith('/contents'):
```

Comment:
Hmm, nope. We add '/contents' over in adapters, and take it away here. The ID can actually be almost anything (though I think I may have added code that enforces that the ident_hash part has to be uuid@ver ) Also, smallest possible change here, to get it out and released.